src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -76,7 +82,9 @@ export default function AnnotationsControls({
                 isEnabled={showDrawing}
                 mode={AnnotationMode.DRAWING}
                 onClick={handleModeClick}
+                setWasClosedByUser={setWasClosedByUser}
```

Comment:
Does `setWasClosedByUser` need to be passed down or can the `tooltipFlowAnnotationsExperience` be modified here to include `setWasClosedByUser` so that it can be passed transparently in the child component